pfcpiface/ip_pool.go

```diff
@@ -0,0 +1,55 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright(c) 2020 Intel Corporation
+
+package main
+
+import (
+	"errors"
+	"log"
+	"net"
+
+	"github.com/wmnsk/go-pfcp/ie"
+)
+
+type ip_pool struct {
+	free_pool []string
+}
+
+func needAllocIp(ueIPaddr *ie.UEIPAddressFields) bool {
+	if has2ndBit(ueIPaddr.Flags) && !has5thBit(ueIPaddr.Flags) {
+		return false
+	}
+	return true
+}
```

Comment:
move this to parse-pdr.go itself? that way this file will purely be IP allocation related.

Reply:
done